pkg/crc/services/dns/dns.go

```diff
@@ -49,10 +49,10 @@ func setupDnsmasq(serviceConfig services.ServicePostStartConfig) error {
 	}
 
 	// Remove the dnsmasq container if it exists during the VM stop cycle
-	_, _, _ = serviceConfig.SSHRunner.Run("sudo podman rm -f dnsmasq")
+	_, _, _ = serviceConfig.SSHRunner.Run("sudo podman rm -f dnsmasq-crc")
```

Comment:
Maybe `crc-dnsmasq`? must the container name match the systemd service file name? snc/the bundle most likely needs an update?